Glyssen/Dialogs/VoiceActorInformationDlg.cs

```diff
@@ -16,5 +18,69 @@ public VoiceActorInformationDlg()
 		{
 			InitializeComponent();
 		}
+
+		private void RemoveSelectedRows(bool confirmWithUser)
+		{
+			bool deleteConfirmed = !confirmWithUser;
+
+			if (confirmWithUser)
+			{
+				deleteConfirmed = MessageBox.Show("Are you sure you want to delete these rows?", "Confirm", MessageBoxButtons.YesNo) == System.Windows.Forms.DialogResult.Yes;
```

Comment:
Any message which gets displayed to the user needs to be wrapped in LocalizationManager.GetString(...).  Better yet, see if you can make use of the dialog in libpalaso.  I think it is called something like recycle*dialog or delete*dialog.